zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/authorization/RoleRecord.java

```diff
@@ -18,6 +20,9 @@ public class RoleRecord extends UnifiedRecordValue implements RoleRecordValue {
   private final LongProperty roleKeyProp = new LongProperty("roleKey", -1L);
   private final StringProperty nameProp = new StringProperty("name", "");
   private final LongProperty entityKeyProp = new LongProperty("entityKey", -1L);
+  private final EnumProperty<EntityType> entityTypeProp =
+      new EnumProperty<>("entityType", EntityType.class, EntityType.USER);
```

Comment:
🔧 I think it'd be good to have an `EntityType.UNSPECIFIED` we can use as default, so we don't accidentally store entities with the wrong entity type.

Reply:
❌ We must declare this property in the constructor and in the wrap method.